### PR TITLE
Tools: autotest.py: factor write_fullresults

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: Tools/autotest/autotest.py [hunk 1]
```diff
@@ -328,40 +328,31 @@ def write_fullresults():
...
-    results.addglob("ArduCopter core", 'ArduCopter.core')
-    results.addglob("ArduCopter elf", 'ArduCopter.elf')
+
+    # results common to all vehicles:
+    vehicle_files = [ ('%(vehicle)s build log', '%(vehicle)s.txt'),
+                      ('%(vehicle)s code size', '%(vehicle)s.sizes.txt'),
+                      ('%(vehicle)s stack sizes', '%(vehicle)s.framesizes.txt'),
+                      ('%(vehicle)s defaults', 'default_params/%(vehicle)s-defaults.parm'),
+                      ('%(vehicle)s core', '%(vehicle)s.core'),
+                      ('%(vehicle)s ELF', '%(vehicle)s.elf'),
```
Comment (OXINARF) [inline Tools/autotest/autotest.py]: Nitpicking: using `{vehicle}` which is the new Python style looks better (and using `.format` later too).

Comment (peterbarker) [reply Tools/autotest/autotest.py]: Hmph.  I changed it away from that :-)

Moved it back...

File: Tools/autotest/autotest.py [hunk 2]
```diff
@@ -328,40 +328,31 @@ def write_fullresults():
...
+    ]
+    vehicle_globs = [('%(vehicle)s log', '%(vehicle)s-*.BIN'),
+                     ]
```
[the remark this answers is not in the document]
Comment (peterbarker) [reply Tools/autotest/autotest.py]: Done.

**Comment by peterbarker** [comment]
@OXINARF Changes made.

**Comment by OXINARF** [comment]
Merged, thanks!

**Pull request merged**